ui/src/components/admin/Notifications/NotificationToaster.js

```diff
@@ -103,15 +105,20 @@ const renderIcon = type => {
 };
 
 const NotificationToaster = () => {
```

Comment:
Changes here are just:
- duplicate `import_____`
- replace `import` with `publish`

To extend the NotificationsToaster to support and display the progress banner for publish progress